lnbits/config.py

```diff
@@ -0,0 +1,62 @@
+import importlib
+import json
+from os import getenv, path
+from typing import List, Optional
+
+from pydantic import BaseSettings, Field, validator
+
+wallets_module = importlib.import_module("lnbits.wallets")
+wallet_class = getattr(    
+    wallets_module, getenv("LNBITS_BACKEND_WALLET_CLASS", "VoidWallet")
+)
+
+WALLET = wallet_class()
+
+def list_parse_fallback(v):
+    try:
+        return json.loads(v)
+    except Exception as e:
+        return v.replace(' ','').split(',')
+
+class Settings(BaseSettings):
+    # users
+    admin_users: List[str] = Field(default_factory=list, env="LNBITS_ADMIN_USERS")
+    allowed_users: List[str] = Field(default_factory=list, env="LNBITS_ALLOWED_USERS")
+    admin_ext: List[str] = Field(default_factory=list, env="LNBITS_ADMIN_EXTENSIONS")
+    disabled_ext: List[str] = Field(default_factory=list, env="LNBITS_DISABLED_EXTENSIONS")
+    funding_source: str = Field(default="VoidWallet", env="LNBITS_BACKEND_WALLET_CLASS")
+    # ops
+    data_folder: str = Field(default=None, env="LNBITS_DATA_FOLDER")
+    database_url: str = Field(default=None, env="LNBITS_DATABASE_URL")
+    force_https: bool = Field(default=True, env="LNBITS_FORCE_HTTPS")
+    service_fee: float = Field(default=0, env="LNBITS_SERVICE_FEE")
+    hide_api: bool = Field(default=False, env="LNBITS_HIDE_API")
+    denomination: str = Field(default="sats", env="LNBITS_DENOMINATION")
+    # Change theme
+    site_title: str = Field(default="LNbits", env="LNBITS_SITE_TITLE")
+    site_tagline: str = Field(default="free and open-source lightning wallet", env="LNBITS_SITE_TAGLINE")
+    site_description: str = Field(default=None, env="LNBITS_SITE_DESCRIPTION")
+    default_wallet_name: str = Field(default="LNbits wallet", env="LNBITS_DEFAULT_WALLET_NAME")
+    theme: List[str] = Field(default="classic, flamingo, mint, salvador, monochrome, autumn", env="LNBITS_THEME_OPTIONS")
```

Comment:
This fails when no value is specified because the default is not a Python list.

```suggestion
    theme: List[str] = Field(default=["classic", "flamingo", "mint", "salvador", "monochrome", "autumn"], env="LNBITS_THEME_OPTIONS")
```

Making it a list like this fixed it for me